sdks/python/apache_beam/io/fileio.py

```diff
@@ -606,6 +606,29 @@ def size_in_bytes(path):
     else:
       return os.path.getsize(path)
 
+  @staticmethod
+  def size_of_files_in_glob(path, file_names):
```

Comment:
I think it's better to make 'file_names' parameter optional. If specified we will try to ensure that all files mentioned in the list are accounted for. We can mention that this is to support file-systems that are only eventually consistent.